### ISSUE TITLE
Supercell dialog testLabel

### ISSUE THREAD

**Issue by Michel-Heinz** [Molara-Lab/Molara#325]
In the supercell dialog, there is a random textLabel floating around with the message textLabel

**Comment by adrianusler** [comment]
Yes, I am aware of this awkward bit. I wanted to implement that this tells you how many atoms are going to be generated (combined with a warning if this number becomes dangerously large).

**Comment by adrianusler** [comment]
I have implemented this in the branch feature/326-snapshot-custom-resolution.
